<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>PaintWidget</class>
    <widget class="QWidget" name="PaintWidget">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>960</width>
                <height>640</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>DrawingBoard</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,9">
            <property name="spacing">
                <number>0</number>
            </property>
            <property name="leftMargin">
                <number>0</number>
            </property>
            <property name="topMargin">
                <number>0</number>
            </property>
            <property name="rightMargin">
                <number>0</number>
            </property>
            <property name="bottomMargin">
                <number>0</number>
            </property>
            <item>
                <layout class="QVBoxLayout" name="paint_layout" stretch="1,1,1,1,1,1">
                    <property name="spacing">
                        <number>2</number>
                    </property>
                    <property name="leftMargin">
                        <number>0</number>
                    </property>
                    <property name="topMargin">
                        <number>0</number>
                    </property>
                    <property name="rightMargin">
                        <number>0</number>
                    </property>
                    <property name="bottomMargin">
                        <number>0</number>
                    </property>
                    <item>
                        <widget class="QToolButton" name="undo_btn">
                            <property name="sizePolicy">
                                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                                    <horstretch>0</horstretch>
                                    <verstretch>0</verstretch>
                                </sizepolicy>
                            </property>
                            <property name="styleSheet">
                                <string notr="true">background-color: rgb(196, 160, 0);
                                    color: rgb(255, 255, 255);
                                </string>
                            </property>
                            <property name="text">
                                <string>undo</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QToolButton" name="redo_btn">
                            <property name="sizePolicy">
                                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                                    <horstretch>0</horstretch>
                                    <verstretch>0</verstretch>
                                </sizepolicy>
                            </property>
                            <property name="styleSheet">
                                <string notr="true">background-color: rgb(143, 89, 2);
                                    color: rgb(255, 255, 255);
                                </string>
                            </property>
                            <property name="text">
                                <string>redo</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QToolButton" name="dot_btn">
                            <property name="sizePolicy">
                                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                                    <horstretch>0</horstretch>
                                    <verstretch>0</verstretch>
                                </sizepolicy>
                            </property>
                            <property name="text">
                                <string/>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QToolButton" name="color_btn">
                            <property name="sizePolicy">
                                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                                    <horstretch>0</horstretch>
                                    <verstretch>0</verstretch>
                                </sizepolicy>
                            </property>
                            <property name="text">
                                <string/>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QToolButton" name="reset_btn">
                            <property name="sizePolicy">
                                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                                    <horstretch>0</horstretch>
                                    <verstretch>0</verstretch>
                                </sizepolicy>
                            </property>
                            <property name="styleSheet">
                                <string notr="true">background-color: rgb(78, 154, 6);
                                    color: rgb(255, 255, 255);
                                </string>
                            </property>
                            <property name="text">
                                <string>reset</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QToolButton" name="ocr_btn">
                            <property name="sizePolicy">
                                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                                    <horstretch>0</horstretch>
                                    <verstretch>0</verstretch>
                                </sizepolicy>
                            </property>
                            <property name="styleSheet">
                                <string notr="true">background-color: rgb(92, 53, 102);
                                    color: rgb(255, 255, 255);
                                </string>
                            </property>
                            <property name="text">
                                <string>OCR</string>
                            </property>
                        </widget>
                    </item>
                </layout>
            </item>
            <item>
                <widget class="QWidget" name="container" native="true"/>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</ui>
